test/hash.cxx

```diff
@@ -7,6 +7,52 @@
 #include <substrate/hash>
 #include <catch2/catch_test_macros.hpp>
 
+TEST_CASE("murmur128", "[hash]")
+{
+	const std::string empty{};
+	const std::array<uint8_t, 1> zero{{0x0}};
+	const std::string example{"The quick brown fox jumps over the lazy dog"};
+	const std::string example2{"The quick brown fox jumps over the lazy dog."};
 
+	const auto pair1{substrate::murmur128(empty, 0x900DBEEF)};
```

Comment:
`UINT32_C()`?

Reply:
I better factor the seed out and wrap it with the macro.